src/main/scala/nl/biopet/tools/refflatsstats/RefflatStats.scala

```diff
@@ -271,4 +271,161 @@ object RefflatStats extends ToolCommand[Args] {
          "IntronOutput.tsv"
        )}
      """.stripMargin
+
```

Comment:
Could you place all of this in a separate file? So we have a separate file with only picard code. That will avoid confusion later.